concepts/Common/Distribution Channels/05 CDN/05 CDN.md

```diff
@@ -2,6 +2,10 @@ DevExtreme provides the following files via CDN:
 
 ## Scripts
 
+    <!-- Scripts for the Diagram and Gantt components -->
+    <script src="https://cdn3.devexpress.com/jslib/minor_20_2/js/dx-diagram.js"></script>
+    <script src="https://cdn3.devexpress.com/jslib/minor_20_2/js/dx-gantt.js"></script>
```

Comment:
You need to move these links up within their categories (Production files and Development files), not separate them into a third category.